orca/scripts/install_orca.sh

```diff
@@ -11,7 +11,7 @@ function main {
 
 	mkdir -p /opt/liferay/orca
 
-	cd /opt/liferay/orca
+	cd /opt/liferay/orca || exit 1
```

Comment:
Can you please move all of these to lcd as I showed this morning?

Reply:
Done.